src/atoms/View/__tests__/View.web.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
```

Comment:
You should go ahead and create a helper function called ```renderWithTheme``` you can refer what we've done for native since you'll need all your stories to have the ```ThemeContext``` from ```styled-components```

Reply:
done